src/components/InfoPageTemplate.js

```diff
@@ -26,7 +19,13 @@ export default function InfoPageTemplate({ id }) {
         price: '₩27,000',
     };
 
-    const renderLabels = Object.keys(sampleData) //hard coded for now, but eventually will do based on id
+    useEffect(() => {
+        axios.get(`http://43.200.193.130:4040/api/books/${id}`).then((res) => {
```

Comment:
```javascript
const instance = axios.create({
  baseURL: 'https://some-domain.com/api/',
  timeout: 1000,
  headers: {'X-Custom-Header': 'foobar'}
});
```


페이지 전체적으로 API baseUrl 이나 여러 axios 기본 설정이 반복되고 있어서 이건 따로 instance를 선언해서 작성하시면 좋을 것 같습니다. 아래 문서 참고하셔서 작업하시면 좋을 것 같아요~
[axios.create](https://axios-http.com/docs/instance)

> The API baseUrl and various axios preferences are repeated throughout the page, so it would be good to declare an instance and write it. I think it would be good to refer to the document below and work on it